indico/modules/events/contributions/client/js/PublicationSwitch.jsx

```diff
@@ -25,87 +27,54 @@ export default function PublicationSwitch({eventId}) {
     return null;
   }
 
+  const trigger = (
+    <Checkbox
+      label={published ? Translate.string('Published') : Translate.string('Draft')}
+      toggle
+      onClick={() => setModalOpen(true)}
+      checked={published}
+      disabled={saving}
+    />
+  );
+
+  const header = published
+    ? Translate.string('Set contribution list in draft mode')
+    : Translate.string('Publish contributions');
+
+  const body = published ? (
+    <>
+      <p>
+        <Translate>
+          Are you sure you want to change the contribution list back to draft mode?
+        </Translate>
+      </p>
+      <p>
+        <Translate>By doing so the following menu items won't be accessible:</Translate>
+      </p>
+    </>
+  ) : (
+    <>
+      <p>
+        <Translate>Are you sure you want to publish the contribution list?</Translate>
+      </p>
+      <p>
+        <Translate>By doing so the following menu items will be accessible:</Translate>
+      </p>
+    </>
+  );
```

Comment:
Why not move this into PublicationModal as well?

Reply:
Also moved the header in as well